modules/end-user-guide/partials/proc_configuring-the-workspace-and-adding-tooling.adoc

```diff
@@ -18,8 +18,12 @@
 
 To add plug-ins to your workspace:
 
-. Click the *Plugins* tab.
-. Enable the plug-in that you want to add and click the btn:[Save] button.
+. Click the *Devfile* tab.
+. Add the `chePlugin` component that you want to add and click the btn:[Save] button.
++
+NOTE: Use completion feature to see available plugins(can be triggered by Ctrl+Space hotkey)
```

Comment:
Probably here we should specify MacOS specific hotkey as well, like Command+Space?

Reply:
Should you use https://asciidoctor.org/docs/user-manual/#keyboard-shortcuts also ?